bin/build

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
+# Uncomment the following line while debugging
```

Comment:
I think we can just add the `x` to the `set` command above when debugging, no need for commented code